src/lib/spinel/spinel_encoder.hpp

```diff
@@ -676,6 +676,14 @@ class Encoder
      */
     otError ResetToSaved(void);
 
+    /**
+     * Clear NCP buffer on reset command.
+     *
+     * @retval OT_ERROR_NONE            Successfully reset done
+     *
+     */
+    otError ClearNcpBuffer(void);
```

Comment:
Change to `void`? since error is always `OT_ERROR_NONE`?

Reply:
done